src/components/plotly/pie/index.js

```diff
@@ -42,15 +42,17 @@ Pie.propTypes = {
   donut: PropTypes.bool,
   showPercentage: PropTypes.bool,
   showLabelName: PropTypes.bool,
-  extra: PropTypes.object,
+  textinfo: PropTypes.string,
+  hole: PropTypes.number,
   ...plotlyPropTypes,
 }
 
 Pie.defaultProps = {
   donut: false,
   showPercentage: true,
   showLabelName: false,
-  extra: {},
+  textinfo: null,
+  hole: null,
```

Comment:
Wouldn't be more correct` hole: 0` because `hole` is a number?

Reply:
perhaps `0.4` to honor existing behavior (so backward compatible). but the additional default depends on `donut` prop too.

therefore the choice of `null` and use of the nullish coalescing operator (`??`)